db/migrate/20180410104202_create_campaigns.rb

```diff
@@ -0,0 +1,16 @@
+class CreateCampaigns < ActiveRecord::Migration[5.1]
+  def change
+    create_table :campaigns do |t|
+      t.string :title
+      t.text :description
+      t.string :status
+      t.integer :current_amount
```

Comment:
I think you need to add default value 0 to current_amount

Reply:
@imysheniuk change type to decimal for money fields

Reply:
@imysheniuk ?????